apis/r/R/TileDBGroup.R

```diff
@@ -75,10 +75,8 @@ TileDBGroup <- R6::R6Class(
         for (member in private$.member_cache) {
           if (!is.null(member$object)) {
             if (member$object$is_open()) {
-              spdl::debug("Openchi {} '{}'", member$object$class(), member$object$uri)
               member$object$close()
             } else {
-              spdl::debug("Clsdchi {} '{}'", member$object$class(), member$object$uri)
```

Comment:
_Technically_  you could also keep them, or downgrade them to `spdl::trace(...)` as they only trigger when the level is set...

Reply:
@eddelbuettel agreed but (a) they're redundant; (b) I really really did not intend to keep them

Reply:
And they do slow things down even if at marginal levels so removal is good too. It's all good :grinning: